tests/libinfra/certificates.go

```diff
@@ -40,3 +49,40 @@ func ContainsCrt(bundle []byte, containedCrt []byte) bool {
 	}
 	return attached
 }
+
+func GetBundleFromConfigMap(configMapName string) ([]byte, []*x509.Certificate) {
+	virtClient := kubevirt.Client()
+	configMap, err := virtClient.CoreV1().ConfigMaps(flags.KubeVirtInstallNamespace).Get(context.Background(), configMapName, v1.GetOptions{})
```

Comment:
can we get the context from outside? (add ctx parameter to the function?)

Reply:
Added another commit to do this